Makefile

```diff
@@ -0,0 +1,8 @@
+MICROOVN_SNAP_PATH=$(CURDIR)/microovn.snap
+build:
+	@echo "Building the snap"
+	@snapcraft pack --use-lxd -o $MICROOVN_SNAP_PATH
+
+func-tests:
```

Comment:
`func-tests` does not ring too well with me, maybe use a target name starting with `check` to be in line with our upstreams?

Add the .snap file as a dependency so that the build is done automatically?
```make
check-system: $(MICROOVN_SNAP_PATH)
```